jogo42-cb/main.c

```diff
@@ -76,6 +81,16 @@ int main(void)
         gs->cam.offset.x += GetScreenWidth() / 2;
         gs->cam.offset.y += GetScreenHeight() / 2;
         gs->cam.target = gs->jog.pos;
+        // Atualizar hp do inimigo
```

Comment:
Era pra ter uma linha vazia separando isso do código de cima
```suggestion

        // Atualizar hp do inimigo
```